src/mock/recorder.cpp

```diff
@@ -403,6 +403,19 @@ namespace librealsense
                 select_api_version.bind(1, section_id);
                 select_api_version.bind(2, API_VERSION_KEY);
                 auto api_version = select_api_version()[0].get_string();
+                const int ver_size = 3;
+                int section_version[ver_size];
+                int min_version[ver_size];
+                std::sscanf(api_version.c_str(), "%d.%d.%d", &section_version[0], &section_version[1], &section_version[2]);
+                std::sscanf(min_api_version.c_str(), "%d.%d.%d", &min_version[0], &min_version[1], &min_version[2]);
+                for (int i = 0; i < ver_size; i++)
```

Comment:
What if invalid string format will be passed?

Reply:
throw exception in case of bad format
done